scripts/release.sh

```diff
@@ -58,9 +66,34 @@ function tag_images_in_yamls() {
   done
 }
 
+function tag_images_in_yamls_acr() {
+  [[ -z ${TAG} ]] && return 0
+  local SRC_DIR="${GOPATH}/src/"
+  echo "Tagging any images under '${KO_DOCKER_REPO}' with ${TAG}"
+  for file in $@; do
+    [[ "${file##*.}" != "yaml" ]] && continue
+    echo "Inspecting ${file}"
+    for image in $(grep -o "${KO_DOCKER_REPO}/[0-9a-z\./-]*@sha256:[0-9a-f]*" ${file}); do
+      local dest_image=$(echo ${image} | cut -d"/" -f2 | cut -d"@" -f1)
+      echo tagging ${dest_image}:${TAG}
+      az acr import -n ${AZ_ACR_NAME} --source ${image} -t ${dest_image}:${TAG}
```

Comment:
Does the repo / dev environment already have az cli as a dependency? Do you need to update docs to account for this?

Reply:
good point! added notes to readme